odp_sdk/tabular_storage_client.py

```diff
@@ -17,15 +17,28 @@
 try:
     from pandas import DataFrame
 except ImportError:
-    print("Pandas not installed. DataFrame support will not be available.")
+    DataFrame = ImportError
+    warn("Pandas not installed. DataFrame support will not be available.")
 
 
 class OdpTabularStorageClient(BaseModel):
     http_client: OdpHttpClient
+    """HTTP-request client"""
+
     tabular_storage_endpoint: str = "/data"
+    """Endpoint appended to base URL in order to interact with the ODP data-APIs"""
+
+    pagination_size: int = 10_000
+    """List-limit when paginating"""
+
+    chunked_encoding_page_size: int = 1_000_000
+    """List-limit when using chunked encoding"""
 
-    def __init__(self, **data):
-        super().__init__(**data)
+    paginate_by_default: bool = False
+    """Whether to paginate by default"""
+
+    page_size_align_factor: int = 1000
+    """Page size alignment factor for pagination - ie. page size must be a multiple of this value"""
```

Comment:
why this limitation? for protected sea the UI switched to a limit of few rows since they are so big.
also I don't think we have limitation on the backend, right?

Reply:
we do :)